DTS-STN/Service-Canada-Labs
# Project page updates

# [Create Explore other projects section](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=215242)

Description of the task you completed and/or features added. If it is a visual component or feature,
add a screenshot(s) of it.

## Test Instructions

1. yarn dev
2. go to benefits navigator page
3. examine explore other projects section (check responsiveness)
4. check out component code to see if there is any changes needed
5. yarn storybook
6. check organisms/exploreProjects entry
